tests/common/output/test_baseline.py

```diff
@@ -0,0 +1,43 @@
+import argparse
```

Comment:
as you want to affect also the order of the resources in some specific file, aren't better we that at least one of the files will continue one then 1 resource, while yor new logic will indeed affect the order?

Reply:
yeah good point 😄